lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -288,41 +288,84 @@ class _SanitizedDetails {
 class _ButtonSanitizer {
   int _pressedButtons = 0;
 
-  // Transform html.PointerEvent.buttons to Flutter's PointerEvent buttons.
+  /// Transform [html.PointerEvent.buttons] to Flutter's PointerEvent buttons.
```

Comment:
At the moment, I don't think we generate docs. But I prefer to use dartdocs anyway to keep the documentation style consistent with the rest of the code base, and because it works better inside the IDE (highlighting and `cmd+click`).